src/api/undeploy/index.js

```diff
@@ -25,18 +25,18 @@ const undeploy = {
         },
         {
           method: 'DELETE',
-          path: '/undeploy-service/{imageName}/all',
+          path: '/scale-to-0/{imageName}',
           ...undeployServiceFromAllEnvironmentController
         },
         {
           method: 'DELETE',
-          path: '/undeploy-service/{imageName}/{environment}',
+          path: '/scale-to-0/{imageName}/{environment}',
```

Comment:
feels like `scale-to-zero` would read a little better?

Reply:
already changed that locally